services/clusters-keeper/src/simcore_service_clusters_keeper/core/settings.py

```diff
@@ -50,6 +51,21 @@ class Config(EC2Settings.Config):
         }
 
 
+class ClustersKeeperSSMSettings(SSMSettings):
+    class Config(SSMSettings.Config):
+        env_prefix = CLUSTERS_KEEPER_ENV_PREFIX
```

Comment:
I see, so this way you can automagically add the prefix to a "standard" names of `SSMSettings`. Nice